mockery/walker.go

```diff
@@ -123,7 +125,9 @@ func (this *GeneratorVisitor) VisitWalk(ctx context.Context, iface *Interface) e
 
 	if this.InPackage {
 		pkg = filepath.Dir(iface.FileName)
-	} else {
+	} else if this.PackageNamePrefix != "" {
+		pkg = fmt.Sprintf("%s%s", this.PackageNamePrefix, iface.Pkg.Name())
```

Comment:
What should the behavior be if `PackageNamePrefix` and `PackageName` are both specified? I think the way this is isn't super clear, because if you specify `PackageNamePrefix`, `PackageName` becomes ignored.